lib/rubocop/cop/style/method_missing.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop check for use of method missing
+      class MethodMissing < Cop
+        MSG = 'Avoid using `method_missing`.'.freeze
+
+        def on_def(node)
```

Comment:
I guess there could also be singleton `method_missing` methods that we should report. You can use the `OnMethodDef` mixin.